jobs/2021-11-30_gnuradio.yaml

```diff
@@ -0,0 +1,23 @@
+entity: GNU Radio
+title: GNU Radio Expert
+url: https://www.gnuradio.org/news/2021-11-30-numfocus-jobs/
+percentTime: Part
+percentOSS: 100
+deadline: 2022-04-01  # Application deadline
+expires: 2022-04-01   # Date when post is removed from board
+location: Remote
+description: |
+ GNU Radio has recently won a small grant from NumFOCUS to help improve GNU Radio’s tutorials, as they play a major role in helping new users get through the steep learning curve. We are looking to hire a part-time contractor to perform the work of one GNU Radio expert. It is anticipated that this work will be performed during Q1 of 2022. Please email jobs@gnuradio.org if you are interested.
+ 
+ ## $3600 total, $60 per hour, 60 hours total spread across at least 1 month (max 3 months)
+
+##Tasks: (tackle one or more of the following tasks)
```

Comment:
I'm not at a computer right now, otherwise I'd fix it for you, but the entire paragraph needs to be indented otherwise it's not.

Reply:
I believe I have fixed the indentation! Thanks for pointing it out :)